schemas/chemotherapy.json

```diff
@@ -50,18 +50,24 @@
       }
     },
     {
-      "name": "chemotherapy_drug_name",
-      "description": "Name of agent or drug administered to patient as part of the chemotherapy treatment regimen.",
+      "name": "drug_rxnormid",
```

Comment:
Since this is not a pk, we need a new PK to handle unique records check and validation, we need a `submitter_therapy_id` in the therapy files.

Reply:
this is a blocker now, so confirm this ASAP please.